style.css

```diff
@@ -13,51 +13,98 @@ body {
   display: flex;
 }
 
+.button {
+  margin-top: 20px;
+}
+
 .logo-img {
   width: 100%;
 }
 
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
 .hamburger {
-  width: 45px;
-  height: 30px;
-  display: flex;
-  align-items: center;
-  margin-top: 20px;
+  display: block;
+  margin:24px 24px 0 0;
 }
 
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
```

Comment:
- [ ] Please add the correct value for the margin-top of your header in the desktop view to match the given design.

![image](https://user-images.githubusercontent.com/68796232/168612770-78574abb-8592-4717-8b87-e1e9aff247eb.png)